### ISSUE TITLE
The code is incomplete and the data set is missing

### ISSUE THREAD

**Issue by fortitude94deng** [pytorch/pytorch#21249]
Hello, have you ever encountered the pytorch official website fine-tuning Maskrcnn, less code, and the data set mask in pedestrian detection is all black，In references/detection/, we have a number of helper functions to simplify training and evaluating detection models. Here, we will use references/detection/engine.py, references/detection/utils.py and references/detection/transforms.py. Just copy them to your folder and use them here.there is less code

**Comment by soumith** [comment]
be more descriptive and reopen issue in https://github.com/pytorch/vision

**Comment by fortitude94deng** [comment]
thanks




On 06/02/2019 00:26, Soumith Chintala wrote:

Closed #21249.
